src/bot/commands/color.js

```diff
@@ -0,0 +1,66 @@
+import debug from 'debug';
+import parseColor from 'parse-color';
+import dedent from 'dedent-js';
+import { RichEmbed } from 'discord.js';
+
+const log = debug('Color');
+
+export const info = {
+  name: 'color',
+  description: 'Lookup information about a color.',
+  usage: dedent`
+    \`\`\`@Aquarius color <name>\`\`\`
+    Name can be one of the following formats:
+    * #hex
+    * keyword
+    * hsl(x, y, z)
+    * rgba(w, x, y, z)
+  `,
+};
+
+/** @type {import('../../typedefs').Command} */
+export default async ({ aquarius, analytics }) => {
+  aquarius.onCommand(/^color (?<name>.+)$/i, async (message, { groups }) => {
+    log(`Looking up color ${groups.name}`);
+
+    let color;
+
+    // Check for hex without the # sign
+    if (
+      (groups.name.length === 3 || groups.name.length === 6) &&
+      !groups.name.startsWith('#')
+    ) {
+      color = parseColor(`#${groups.name}`);
+    } else {
+      color = parseColor(groups.name);
+    }
+
+    if (!color.hex) {
+      message.channel.send(
+        "Sorry, I wasn't able to parse that color! I only can lookup colors formatted as a `#0A0A0A` hex, a `rebeccapurple` keyword, in `hsl(50, 210, 50)` hsl format, or in `rgba(15,53,242,30%)` rgba format."
+      );
+    } else {
+      log(
+        `http://www.thecolorapi.com/id?format=svg&named=false&hex=${color.hex.substring(
+          1
+        )}`
+      );
+      const embed = new RichEmbed()
+        .setTitle(groups.name)
+        .setThumbnail(
+          `http://singlecolorimage.com/get/${color.hex.substring(1)}/350x350`
+        )
+        .setColor(parseInt(color.hex.substring(1), 16))
```

Comment:
There's a new method for this - also should see if substring or replace is faster